pkg/operator/ceph/cluster/osd/spec.go

```diff
@@ -210,11 +210,19 @@ func (c *Cluster) makeDeployment(osdProps osdProperties, osd OSDInfo, provisionC
 		// add the PVC size to the pod spec so that if the size changes the OSD will be restarted and pick up the change
 		envVars = append(envVars, v1.EnvVar{Name: "ROOK_OSD_PVC_SIZE", Value: osdProps.pvcSize})
 
-		// Append tuning flag if necessary
+		// Append slow tuning flag if necessary
 		if osdProps.tuneSlowDeviceClass {
-			err := c.osdRunFlagTuningOnPVC(osd.ID)
+			err := c.osdRunFlagSlowTuningOnPVC(osd.ID)
 			if err != nil {
-				return nil, errors.Wrapf(err, "failed to apply tuning on osd %q", strconv.Itoa(osd.ID))
+				return nil, errors.Wrapf(err, "failed to apply slow tuning on osd %q", strconv.Itoa(osd.ID))
+			}
+		}
+
+		// Append fast tuning flag if necessary
+		if osdProps.tuneFastDeviceClass {
```

Comment:
Should this be an "else if"? Not sure we want to allow slow and fast devices to both be tuned at the same time. It makes me wonder if the `tuneDeviceClass` should be an enum, but that would be a breaking change, so we better avoid that.

Reply:
Done.